eclass/dict.eclass

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: dict.eclass
+# @MAINTAINER:
+# maintainer-needed@gentoo.org
+# @AUTHOR:
+# Original author: Seemant Kulleen
+# @BLURB: Ease the installation of dict translation dictionaries
+# @DESCRIPTION:
+# This eclass exists to ease the installation of dict translation
+# dictionaries.  The only variables which need to be defined in the actual
+# ebuilds is DICTS if the package ships more than one dictionary which
+# cannot be determined from ${PN}.
+
+# @ECLASS-VARIABLE: DICTS
+# @DESCRIPTION:
+# Array of dictionary file names (foo.dict.dz and foo.index) to be installed
+# during dict_src_install
+
+if [[ -z ${_DICT_ECLASS} ]]; then
+_DICT_ECLASS=1
+
+case ${EAPI:-0} in
+	6) ;;
```

Comment:
mayeb also enable EAPI 7

Reply:
Ah yeah. I was working on this before it was acceptable upstream. Thanks for that.